### ISSUE TITLE
Allow custom mailables to be rendered without setting a campaign

### ISSUE THREAD

**Issue by jbraband** [spatie/mailcoach-support#147]
I am building out a bunch of custom Mailable classes for a bunch of different email campaigns. While developing the blade templates it is nice to be able to render the Mailable in a browser for testing. This is documented in the Laravel Docs < Mail > Previewing Mailables In The Browser (https://laravel.com/docs/7.x/mail#previewing-mailables-in-the-browser).

However, when my Mailables extend Spatie\Mailcoach\Mails\CampaignMail like is required for use by Mailcoach, the following exception is thrown when preview rendering it in the browser:

> Typed property Spatie\Mailcoach\Mails\CampaignMail::$campaign must not be accessed before initialization

Three ways to get around this:

1. Remove the `extends CampaignMail` while previewing in the browser
2. In the method used by the route, also call $mailable->setCampaign($campaign) and pass in a dummy Campaign model
3. (this is where the Spatie teams comes in) make the public $campaign property of CampaignMail nullable as seen below

```
class CampaignMail extends Mailable
{
    use SerializesModels;

    public ?Campaign $campaign = null;

    .....
}
```

Is there issue elsewhere in the codebase with this change? Perhaps some additional checks to ensure that the $campaign is not null. The $campaign property could also go to protected and add a CampaignMail::getCampaign() that simply returns the protected $campaign property). 

It would just make developing custom mailables a little more straightforward.

**Comment by freekmurze** [comment]
> make the public $campaign property of CampaignMail nullable as seen below

I've made this change and released it as `v2.7.1` 👍

**Comment by jbraband** [comment]
Thanks @freekmurze !
